Lib/Src/CAdcData.cpp

```diff
@@ -14,7 +14,7 @@
 #define ADC_START_BIT        2
 #define ADC_CONVERSION_COEFF ((float)ADC_VDDA / ADC_RES)
 
-CUartCom uart_adc(&huart2);
+CUartCom uart_adc(&huart2, "ADC Data");
```

Comment:
Once this UART branch is approved I will check if CLog is necessary. It's likely that I will delete this in the future.